sourcepoint-ui/TCFv2_banner_amp.css

```diff
@@ -409,7 +409,7 @@ https://github.com/guardian/consent-management-platform/tree/main/sourcepoint-ui
     
     .gu-content {
         border-top: 1px solid rgb(80, 105, 145) !important;
-        max-height: calc(51vh) !important;
+        max-height: calc(100vh) !important; /* See above */
```

Comment:
how does this behave where [`vh` is not supported](https://caniuse.com/?search=vh)?

Reply:
Oh, that’s a good point. I had assumed support was more widespread.

In this case this only applies to AMP, so support is only relevant in mobile browsers. However, we are using similar rules in the main file to prevent from covering the whole page.

Reply:
i think it will still not work in older opera mobile, which is 0.28% of our readers for the last month:

<img src="https://user-images.githubusercontent.com/867233/94552174-5fa0a400-024e-11eb-975e-b2504b289ab2.png" width="501"/>

Reply:
AMP uses `vh` [in its codebase][1] already… How can we test this further?

[1]: https://github.com/ampproject/amphtml/blob/ab581081c76851f9e31e1558c9d06f8b4b4fbc52/extensions/amp-story/1.0/amp-story-consent.css